sklearn/linear_model/_least_angle.py

```diff
@@ -56,23 +56,23 @@ def lars_path(
 
     (1 / (2 * n_samples)) * ||y - Xw||^2_2 + alpha * ||w||_1
 
-    in the case of method='lars', the objective function is only known in
-    the form of an implicit equation (see discussion in [1])
+    in the case of method='lar', the objective function is only known in
+    the form of an implicit equation (see discussion in [1]).
 
     Read more in the :ref:`User Guide <least_angle_regression>`.
 
     Parameters
     ----------
     X : None or array-like of shape (n_samples, n_features)
-        Input data. Note that if X is None then the Gram matrix must be
-        specified, i.e., cannot be None or False.
+        Input data. Note that if X is ``None`` then the Gram matrix must be
+        specified, i.e., cannot be ``None`` or ``False``.
```

Comment:
```suggestion
        Input data. Note that if X is `None` then the Gram matrix must be
        specified, i.e., cannot be `None` or `False`.
```

Reply:
Would you like to have single grave accents everywhere? I put two because that seemed to be the pattern used throughout the remainder of the docstring. Then I'd clean this up for the entire docstring. (I checked the documentation guidelines but they don't seem to mention any convention regarding this.)

Reply:
There was a distinction between single and double backticks at some point but it's no longer the case iirc, so we prefer to use singles today. You don't need to change the ones you're not modifying though to keep the review of this PR simple.

Reply:
Got it, thanks!